src/test/scala/org/apache/spark/sql/kinesis/ShardSyncerSuite.scala

```diff
@@ -29,14 +29,16 @@ class ShardSyncerSuite extends SparkFunSuite with SharedSQLContext {
 
   test("Should error out when failondataloss is true and a shard is deleted") {
```

Comment:
Can we add a unit test for fromCheckpointJson

Reply:
I've added KinesisPositionSuite test.
Please let me know if you want to have some specific test in addition.